grade.php

```diff
@@ -14,6 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
+/**
+ * File to .
```

Comment:
This is for the [Gradebook redirection](https://docs.moodle.org/dev/Gradebook_API#Gradebook_redirection_page).

Reply:
thanks! adjusted that in the comment.